build.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-PLATFORM="${TEXTURE_BUILD_PLATFORM:-platform=iOS Simulator,OS=10.2,name=iPhone 7}"
-SDK="${TEXTURE_BUILD_SDK:-iphonesimulator12.2}"
+echo !!!!
```

Comment:
Do we need these echo statements still or was this just for debugging?

Reply:
just debugging. There is no documentation how to switch Xcode.